### DISCUSSION TITLE
Creating a multiple tag input text area

### DISCUSSION THREAD

**Discussion by praj18** [chakra-ui/chakra-ui#5163] [Q&A] [answered]
Is it possible to create a tag input area, such as the one shown below, where multiple emails can be inputted into the text area, with just ChakraUI?

<img width="550" alt="Screenshot 2021-11-28 at 12 02 51 PM" src="https://user-images.githubusercontent.com/62665365/143729069-ed875ff1-8c11-4ddb-bd7f-badeab0a8130.png">

**Comment by anubra266** [comment, marked as answer]
Check out https://github.com/anubra266/choc-autocomplete

**Reply by praj18** [reply]
Thank you. I'll check it out
